### DISCUSSION TITLE
allure-results folder has .json and container files but no .png files for screenshots

### DISCUSSION THREAD

**Discussion by swatinigam** [Shelex/cypress-allure-plugin#177] [Q&A]
Hello there,

When I try to run allure serve command, I see the report generated as expected but I don't see screenshots for failed tests. Here is my cypress.config.js file:
************************
```
const { defineConfig } = require('cypress');
const allureWriter = require('@shelex/cypress-allure-plugin/writer');

module.exports = defineConfig({
  video: true,
  downloadsFolder: 'cypress/fixtures/downloads',
  defaultCommandTimeout: 25000,
  numTestsKeptInMemory: 0,
  pageLoadTimeout: 25000,
  responseTimeout: 25000,
  chromeWebSecurity: false,
  e2e: {
    setupNodeEvents(on, config) {
      allureWriter(on, config);
      return config;
    },
  },
  env: {
    allure: true,
    allureLogCypress: true,
    allureReuseAfterSpec: true,
    allureResultsPath: './allure-results',
    allureAttachRequests: true,
    allureSkipAutomaticScreenshots: false,
    allureAddVideoOnPass: true,
    allureAddAnalyticLabels: true
  },
});

```
And here are versions for packages I'm using:
```
    "allure-commandline": "^2.20.1",
    "cypress": "^10.8.0",
    "npx": "^10.2.2",
    "@shelex/cypress-allure-plugin": "^2.34.0"

```
I created a very basic test which is bound to fail. I see that the screenshot is sent to the cypress/screenshots and I see the video in the cypress/video folder but I see neither in the report. Attaching a screenshot for both. 
I hope it is something simple that I am missing given that no one else is complaining about screenshots not generating in the allure-results folder.
![Screen Shot 2022-12-22 at 11 04 10 PM](https://user-images.githubusercontent.com/3534976/209268745-29d9d5e3-fe64-4c0a-9488-a59b53be17c3.png)

 
![Screen Shot 2022-12-22 at 11 05 06 PM](https://user-images.githubusercontent.com/3534976/209268848-bdaece80-f355-4d18-b8d7-8c996a794ae6.png)

**Comment by swatinigam** [comment]
@Shelex sorry to bother you. Would you mind taking a look at my config above? I don't see screenshots attached to my report when tests fail. Thank you.

**Reply by AmirAmArA** [reply]
@swatinigam did you use allure writer ?

**Comment by Shelex** [comment]
For your configuration you should have `allureReuseAfterSpec: false`, as no other listeners for this event are registered. That way I was able to get video and screenshot attached.
